pkg/collector/python/datadog_agent.go

```diff
@@ -221,11 +222,21 @@ func lazyInitObfuscator() *obfuscate.Obfuscator {
 }
 
 // ObfuscateSQL obfuscates & normalizes the provided SQL query, writing the error into errResult if the operation
-// fails
+// fails. An optional configuration may be passed to change the behavior of the obfuscator.
 //export ObfuscateSQL
-func ObfuscateSQL(rawQuery *C.char, errResult **C.char) *C.char {
+func ObfuscateSQL(rawQuery, options *C.char, errResult **C.char) *C.char {
+	var sqlCfg traceconfig.SQLObfuscationConfig
+	if options != nil {
+		jl := &jlexer.Lexer{Data: []byte(C.GoString(options))}
+		sqlCfg.UnmarshalEasyJSON(jl)
```

Comment:
What's the reason for including this third-party package? Could we not use the stdlib JSON unmarshaling via https://golang.org/pkg/encoding/json/?

Reply:
This method is called very frequently from the integration and unmarshalling has some performance overhead, so I opted to use the `easyjson` package to optimize this as much as possible. For reference, the benchmarks between using `easyjson` over `encoding/json` is included in the PR body. TLDR; `easyjson` out performs `encoding/json`. I've also written a test to ensure the output is the same when using `easyjson`.

Reply:
Fair enough (and I missed the bottom info on the PR) 👍